test-appimagetool.sh

```diff
@@ -73,13 +73,15 @@ log "remove the default ignore file, and check if an explicitly passed one works
 rm .appimageignore
 echo "to-be-ignored" > ignore
 "$appimagetool" appimagetool.AppDir appimagetool.AppImage --exclude-file ignore || false
-"$appimagetool" -l appimagetool.AppImage | grep -q to-be-ignored && false
+"$appimagetool" -l appimagetool.AppImage | grep -q to-be-ignored && false   <= Isn't this line wrong? Should be like next:
+"$appimagetool" -l appimagetool.AppImage | grep -q ignore && false 
```

Comment:
This line doesn't make sense at all. The ignore file is not in the AppDir and will never end up in the AppImage. It is not necessary. Also, your comment above doesn't make sense any more, as I explained elaborately in #683.